### PR TITLE
Sophos EDR: documentation

### PR DESCRIPTION
**Pull request by squioc**
add the documentation to ingest Sophos EDR events

### CODE REVIEW

File: docs/operation_center/integration_catalog/endpoint/sophos_edr.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+uuid: 3f330d19-fdea-48ac-96bd-91a447bb26bd
+name: Sophos EDR
+type: intake
+
+## Overview
+
+Sophos EDR, monitor, detects and mitigates threats on endpoints.
```
Comment (ka0ula) [inline docs/operation_center/integration_catalog/endpoint/sophos_edr.md:7]:
```suggestion
Sophos EDR monitors, detects and mitigates threats on endpoints.
```

File: docs/operation_center/integration_catalog/endpoint/sophos_edr.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+uuid: 3f330d19-fdea-48ac-96bd-91a447bb26bd
+name: Sophos EDR
+type: intake
+
+## Overview
+
+Sophos EDR monitors, detects and mitigates threats on endpoints.
+This EDR reduces the attack surface and prevent attacks from running with an anti-exploit, an anti-ransomware and advanced control technology.
+
+This setup guide shows how to forward events produced by Sophos EDR to SEKOIA.IO.
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/endpoint/sophos_edr.md:10]:
```suggestion
This setup guide describe how to forward events produced by Sophos EDR to SEKOIA.IO.
```

File: docs/operation_center/integration_catalog/endpoint/sophos_edr.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+{!operation_center/integration_catalog/generated/sophos-edr_do_not_edit_manually.md!}
+
+## Configure
+
+### Credentials
+
+In the Sophos Central Admin, go to `Global Settings > API Credentials Management`.
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/endpoint/sophos_edr.md:18]:
```suggestion
In the Sophos Central Admin console, go to `Global Settings > API Credentials Management`.
```

File: docs/operation_center/integration_catalog/endpoint/sophos_edr.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Create a new token with `Add Credential`. Type a Credential name, select the appropriate role and add a description.
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/endpoint/sophos_edr.md:19]:
```suggestion
Create a new token with `Add Credential`. Complete the name, appropriate role and description.
```

File: docs/operation_center/integration_catalog/endpoint/sophos_edr.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+On the `API credential Summary`, copy the `Client ID` and the `Client Secret`.
+
+### Create the intake
+
+Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format `AWS CloudTrail`.
+
+### Pull events
+
+Get the [Sophos Central SIEM integration script](https://github.com/sophos/Sophos-Central-SIEM-Integration). Configure it according [this guide](https://support.sophos.com/support/s/article/KB-000036372?language=en_US)
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/endpoint/sophos_edr.md:28]:
```suggestion
Get the [Sophos Central SIEM integration script](https://github.com/sophos/Sophos-Central-SIEM-Integration) and configure it by following [this Sophos guide](https://support.sophos.com/support/s/article/KB-000036372?language=en_US).
```

File: docs/operation_center/integration_catalog/endpoint/sophos_edr.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+### Forward events
+
+See our [Ingestion methods page](https://docs.sekoia.io/operation_center/data_collection/ingestion_methods/) to select the appropriate methods to forward your events to sekoia.io according to your needs.
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/endpoint/sophos_edr.md:32]:
```suggestion
See the [Ingestion methods page](https://docs.sekoia.io/operation_center/data_collection/ingestion_methods/) to select the appropriate method to forward your events to SEKOIA.IO according to your needs.
```

> Don’t use "I", "our", "us", "we", use "you".

https://zeroheight.com/6104b1d12/p/859e67-pronouns

> When referring to SEKOIA.IO, always put the name in capital letters. 

https://zeroheight.com/6104b1d12/p/9392ba-capitalization

**Review by PierrickV — COMMENTED** [review]
Thank you for this work ! :+1: 

:warning:  You have an auto-generated file into this PR

File: docs/xdr/features/collect/integrations/endpoint/sophos_edr.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+### Credentials
+
+In the Sophos Central Admin console, go to `Global Settings > API Credentials Management`.
+Create a new token with `Add Credential`. Complete the name, appropriate role and description.
+On the `API credential Summary`, copy the `Client ID` and the `Client Secret`.
+
+### Create the intake
+
+Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format `AWS CloudTrail`.
```
Comment (gaelmuller) [inline docs/xdr/features/collect/integrations/endpoint/sophos_edr.md:24]:
```suggestion
Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a new intake from the format `Sophos EDR`.
```

**Review by gaelmuller — CHANGES_REQUESTED** [review]

**Review by gaelmuller — APPROVED** [review]

**Pull request merged**